src/Symfony/Component/Form/DataMapperInterface.php

```diff
@@ -22,12 +22,12 @@ interface DataMapperInterface
      * The method is responsible for calling {@link FormInterface::setData()}
      * on the children of compound forms, defining their underlying model data.
      *
-     * @param mixed                        $viewData View data of the compound form being initialized
-     * @param FormInterface[]|\Traversable $forms    A list of {@link FormInterface} instances
+     * @param mixed                    $viewData View data of the compound form being initialized
+     * @param FormInterface[]|iterable $forms    A list of {@link FormInterface} instances
```

Comment:
Would `iterable<FormInterface>` be a correct notation here?

Reply:
it's a correct notation, but do we use it in Symfony?